source/index.rst

```diff
@@ -3,12 +3,47 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to open5e
-==================================
+Open5e
+======
 
-*The best open source for the SRD, and the source of the best open content*
+Open5e aims to be the best open-source resource for fifth edition d20 content.
 
-Contents:
+.. rst-class:: hide-mobile
+
+What are we about?
+------------------
+
+.. rst-class:: hide-mobile
+
+In addition to an open, readable, and easily searchable resource for the 5e SRD, we also intend to provide:
```

Comment:
This sounds like we already have it. Instead, maybe "We have lots of new things we'd like to add! At the top of our list are:"